src/edu/ucsd/library/xdre/statistic/analyzer/LogAnalyzer.java

```diff
@@ -31,7 +31,8 @@
  * @author lsitu@ucsd.edu
  */
 public class LogAnalyzer{
-	
+	public static String SPEC_COLL_URL = "http://library.ucsd.edu/speccoll/";
```

Comment:
I'm concerned about the `http` part of this. Very soon we're going to be redirecting/changing all URL's on library to `https`. Do we need to specify `http` specifically here?